zubhub_frontend/zubhub/src/assets/css/index.css

```diff
@@ -596,6 +596,16 @@ html,
   display: none !important;
 }
 
-.MuiInputBase-root #phone{
+.MuiInputBase-root #phone {
   width: 80%;
-}
\ No newline at end of file
+}
+.MuiGrid-root #sixCenter {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
As you told i have defined classes inside the useStyles and added classes instead of id's but the label layout is changing as below and the 


<img width="839" alt="Screenshot 2023-03-10 at 9 42 24 PM" src="https://user-images.githubusercontent.com/88829894/224366593-702af2bc-d3fb-45dc-a02f-b9768ff621be.png">





in order to changes to appear id's are the way because of their specificity or by using `!important` to change line height  inside the `labelClass` which will not effect anything

Reply:
This is what I got with the changes I mentioned, Add new css class and use flexbox with center alignment. Try again.


<img width="965" alt="image" src="https://user-images.githubusercontent.com/2925876/224427936-4ea51813-19f1-4a9d-9214-376072286ada.png">

Reply:
In the screen shot you send above the numbering label and title label are not beside by each other which effects the design won't you agree the actual design i have pinned below have look at it 
<img width="625" alt="Screenshot 2023-03-09 at 1 07 36 AM" src="https://user-images.githubusercontent.com/88829894/224470073-b34d2883-6c25-49e7-b8ac-4b4628b0e73b.png">

Reply:
I missed it but it can be fixed too.
Just the parent needs to be flex
<img width="1125" alt="image" src="https://user-images.githubusercontent.com/2925876/224477991-427db060-85ca-4d52-b480-c3ad27c5521e.png">

Reply:
@kamthamc modified the changes as per request kindly review it

Reply:
@kamthamc  i have modified the changes as per requirements  can you review it thank you for your patience 😄

Reply:
@tuxology No in my approach i have taken that into consideration and made the changes

Reply:
<img width="557" alt="Screenshot 2023-03-13 at 4 17 44 PM" src="https://user-images.githubusercontent.com/88829894/224680337-9aa0d98c-769e-4364-8735-e63a1305c932.png">

Reply:
@tuxology As you can see that the numbers are centred and also they are of same size and there is no need to make any changes further kindly review it and merge the pull request